dev/reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -50,6 +50,28 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - Default value: 0
 - This variable is used to set whether the optimizer executes the optimization operation of unfolding the "in-" subquery.
 
+### tidb_auto_analyze_ratio
+
+- Scope: GLOBAL
+- Default value: 0.5
+- This variable is used to set the threshold value for the automatic updates by `ANALYZE` statements. For the table `tbl`, when the ratio of the number of the modified rows and the total rows is greater than `tidb_auto_analyze_ratio` and the current time is between `tidb_auto_analyze_start_time` and `tidb_auto_analyze_end_time`, TiDB executes the `ANALYZE TABLE tbl` statement in the background to automatically update the statistics of the table.
+
+> **Note:**
+>
+> Only when the `run-auto-analyze` option is enabled in the starting configuration file of TiDB, the `auto_analyze` feature can be triggered.
+
+### tidb_auto_analyze_start_time
+
+- Scope: GLOBAL
+- Default value: 00:00 +0000
+- This variable is used to set the start time of a day, after which the automatic updates by `ANALYZE` statements are allowed.
```

Comment:
This is a little misleading since you are not running the ANALYZE statement directly. Here is my attempt to clarify:
```suggestion
- This variable is used to restrict the time window that the automatic update of statistics is permitted. For example, to only allow automatic statistics updates between 1AM and 3AM, set `tidb_auto_analyze_start_time='01:00 +0000'` and `tidb_auto_analyze_end_time='03:00 +0000'`.
```